tests/system/Config/fixtures/.env

```diff
@@ -4,6 +4,12 @@ SPACED="with spaces"
 
 NULL=
 
+char.expo.foo="exported foo"
+character.export.var=variable
+export char.var=character
+export char.exports=imports
+fruit.export = "banana"
```

Comment:
Before: $_ENV
```
    'char.expo.foo' => string (12) "exported foo"
    'character..var' => string (8) "variable"
    ' char.var' => string (9) "character"
    ' char.s' => string (7) "imports"
    'fruit.' => string (6) "banana"
```
After: $_ENV
```
    'char.expo.foo' => string (12) "exported foo"
    'character.export.var' => string (8) "variable"
    'char.var' => string (9) "character"
    'char.exports' => string (7) "imports"
    'fruit.export' => string (6) "banana"
```

Reply:
Node.js dotenv@16.0.3:
```
  'char.expo.foo': 'exported foo',
  'character.export.var': 'variable',
  'char.var': 'character',
  'char.exports': 'imports',
  'fruit.export': 'banana'
```
It seems that `export` is handled the same way in Node dotenv as in this PR.